src/NServiceBus.Core.Tests/Routing/UnicastRoutingTableTests.cs

```diff
@@ -1,96 +1,92 @@
 ﻿namespace NServiceBus.Core.Tests.Routing
 {
-    using System;
-    using System.Threading.Tasks;
-    using Extensibility;
+    using System.Collections.Generic;
+    using ApprovalTests;
     using NServiceBus.Routing;
     using NUnit.Framework;
 
     [TestFixture]
     class UnicastRoutingTableTests
     {
         [Test]
-        public void When_registering_multiple_static_routes_for_same_type_should_throw_exception()
+        public void When_group_does_not_exist_routes_are_added()
```

Comment:
nitpicking: the used naming conventions here would be "When....Should....". E.g. something like "When_group_does_not_exist_should_add_routes".

Not a blocker, but you'd earn a smiley-sticker for that ;)